crates/cli/src/lib.rs

```diff
@@ -306,23 +317,25 @@ pub fn build_app<'a>() -> Command<'a> {
             .arg(
                 Arg::new(ROC_FILE)
                     .help("The .roc file for an app using the platform")
-                    .allow_invalid_utf8(true)
+                    .value_parser(value_parser!(PathBuf))
                     .required(true)
             )
             .arg(
                 Arg::new(FLAG_TARGET)
                     .long(FLAG_TARGET)
                     .help("Choose a different target")
-                    .default_value(Target::default().into())
-                    .possible_values(Target::iter().map(|target| {
-                        Into::<&'static str>::into(target)
-                    }))
+                    .default_value(Into::<&'static str>::into(Target::default()))
+                    .value_parser(
+                        PossibleValuesParser::new(
+                            Target::iter().map(|target| {
+                                Into::<&'static str>::into(target)
+                            })
+                        ))
                     .required(false),
```

Comment:
Given this is a bit complex, can we define it as a variable at the top and share it between all uses?

Reply:
Done